src/Microsoft.Identity.Abstractions/DownstreamApi/IDownstreamApi.HttpMethods.tt

```diff
@@ -24,13 +28,24 @@ namespace Microsoft.Identity.Abstractions
     public partial interface IDownstreamApi
     {
 <#
+  foreach(string framework in new string[]{  "all", "net8" } )
+  {
+  
+    if (framework == "net8")
+    {
+#>
+
+#if NET8_0_OR_GREATER
+<#
+    }
+
   foreach(string httpMethod in new string[]{  "Get", "Post", "Put", "Patch", "Delete"} )
   {
    if (httpMethod == "Patch")
```

Comment:
(nit) 
```suggestion
   if (httpMethod == "Patch" && framework != "net8")
```

This will remove the nested `#if NETSTANDARD2_1_OR_GREATER || NET` when you are already in the `#if NET8_0_OR_GREATER` condition. That nested `#if` is unnecessary since you are already in .NET 8+.